app/src/main/java/ch/blinkenlights/android/vanilla/CoverCache.java

```diff
@@ -463,6 +468,19 @@ public Bitmap createBitmap(Song song, long maxPxCount) {
 					}
 				}
 
+				if (inputStream == null && (CoverCache.mCoverLoadMode & CoverCache.COVER_MODE_INLINE) != 0)
+				{
+					android.media.MediaMetadataRetriever mmr = new MediaMetadataRetriever();
+					mmr.setDataSource(song.path);
```

Comment:
You imported the class, so you can just do
MediaMetadataRetriever mmr = ....

Reply:
changed